src/framework/global/thirdparty/deto_async/async/internal/abstractinvoker.cpp

```diff
@@ -12,7 +12,7 @@ AbstractInvoker::AbstractInvoker()
 
 AbstractInvoker::~AbstractInvoker()
 {
-    std::lock_guard<std::mutex> lock(m_qInvokersMutex);
+    std::lock_guard lock(m_qInvokersMutex);
     for (QInvoker* qi : m_qInvokers) {
         qi->invalidate();
```

Comment:
Wait, it's still not good: now I'm removing things from `m_qInvokers` while iterating over it! Will fix.